### PR TITLE
Whyderrick fixes

### PR DESCRIPTION
**Pull request by whyderrick**
Bunch of minor changes that I didn't want to email you about one-by-one. 

Most significant changes were these:
- The HTML to PDF generators I used didn't recognize `span class="Section" /> so I replaced them with divs that existed everywhere else.
- Made some changes to Sam Rush's background image that should keep it from overflowing on a page
- Applied the `Section` class a few additional places
- Other minor typographical changes

### CODE REVIEW

File: assets/stylesheets/style.css [hunk 1]
```diff
@@ -49,10 +50,12 @@ div.offset-s {
 
 .rush-x {
 	width: 100%;
+	max-width: 700px;
```
Comment (mdayaram) [inline assets/stylesheets/style.css]: max-width is actually not allowed in `ebook` formats which is why I removed it originally.

Comment (whyderrick) [inline assets/stylesheets/style.css]: Good eye. Thanks!

**Comment by mdayaram** [comment]
Other than the `max-width`, everything looks good.  I can merge it in once you undo the `max-width` bit.

**Comment by whyderrick** [comment]
Should be done. Thanks!

**Comment by mdayaram** [comment]
Merged, and ran the script with the latest updates.

**Comment by whyderrick** [comment]
Thank you!

**Pull request merged**